mitodl/mitxpro
# #1099 Program certificate links and view

#### Pre-Flight checklist


#### What are the relevant tickets?
#1099 

#### What's this PR do?
Adds certificate links for programs on the dashboard page. Also adds program certificate views (the same template handles course and program certificates).

#### How should this be manually tested?
Ensure that a program certificate is generated against any of the user's enrollments. Visit the user's dashboard. The certificate link should appear for the program the same way it appears for courses. Clicking on the link should take you to the certificate in a new tab.

#### Screenshots (if appropriate)
Desktop dashboard
![screencapture-xpro-odl-local-8053-dashboard-2019-09-12-16_01_20](https://user-images.githubusercontent.com/45350418/64782273-36976d80-d57e-11e9-8ca1-78f45711be36.png)

Mobile dashboard
![screencapture-xpro-odl-local-8053-dashboard-2019-09-12-16_01_32](https://user-images.githubusercontent.com/45350418/64782299-4616b680-d57e-11e9-8ac6-37a535a726b2.png)

Desktop program certificate view
![screencapture-xpro-odl-local-8053-certificates-program-e7c8c185-dea6-448e-8888-d81c96baa305-2019-09-12-16_06_58](https://user-images.githubusercontent.com/45350418/64782319-529b0f00-d57e-11e9-877e-665cb5c6b0b3.png)

Mobile program certificate view
![screencapture-xpro-odl-local-8053-certificates-program-e7c8c185-dea6-448e-8888-d81c96baa305-2019-09-12-16_07_15](https://user-images.githubusercontent.com/45350418/64782373-72323780-d57e-11e9-938d-146e43068c63.png)
